cargo-dist/Cargo.toml

```diff
@@ -15,7 +15,8 @@ exclude = [
 ]
 
 [[bin]]
-name = "cargo-dist"
+name = "dist"
+path = "src/main.rs"
```

Comment:
this is necessary because the package name is still cargo-dist ya?

Reply:
That's correct.